### ISSUE TITLE
exon coverage

### ISSUE THREAD

**Issue by stolpeo** [varfish-org/varfish-server#2040]
There has been the request to display the coverage of an exon upon request, or more specific if the coverage of the exons of a gene have at least X coverage, or if an exon has insufficient coverage.

**Comment by stolpeo** [comment]
this information is currently not available in varfish. probably the information should come from the pipeline and somehow be imported into varfish using a new data structure. how and where this information should be made available is up for discussion.

**Comment by holtgrewe** [comment]
I agree, we need to find a solution with either pre-compute for all exon or retrieve online from BAM.
